rumqttd/src/lib.rs

```diff
@@ -92,7 +91,9 @@ impl TlsConfig {
                 .iter()
                 .all(|v| Path::new(v).exists()),
             TlsConfig::NativeTls { pkcs12path, .. } => Path::new(pkcs12path).exists(),
-        }
+        };
+        trace!("Validating TLS config paths with result: {:?}", result);
+        result
```

Comment:
I believe this log should be where the function is actually used, not inside the function! wdyt? also, is this helpful as if paths aren't validated, we return error anyway right??

Reply:
I've changed the traces as follows:


```rust
trace!("Validated certificate paths for server v4.{name}.");
trace!("Validated certificate paths for server v5.{name}.");
trace!("Validated certificate paths for server ws.{name}.");
```

at the caller site

Reply:
I wanted to say that, is the certs aren't valid, we are going to panic, so what's the use of this trace? I believe it's not providing any helpful info. Thoughts?